hw/opentitan/ot_otp_eg.c

```diff
@@ -2564,140 +2540,146 @@ static void ot_otp_eg_reg_write(void *opaque, hwaddr addr, uint64_t value,
 
 static const char *ot_otp_eg_swcfg_reg_name(unsigned swreg)
 {
-#define CASE_SCALAR(_reg_) \
+#define CASE_BYTE(_reg_) \
+    case A_##_reg_: \
+        return stringify(_reg_)
+#define CASE_SUB(_reg_, _sz_) \
+    case A_##_reg_...(A_##_reg_ + (_sz_)): \
```

Comment:
I don't think anything on Darjeeling uses this case either, but perhaps this should be:

```suggestion
    case A_##_reg_...(A_##_reg_ + (_sz_) - 1u): \
```
As for example I would expect `CASE_SUB(EXAMPLE, 3u)` to expand to `A_EXAMPLE...(A_EXAMPLE + 2)` and not `A_EXAMPLE...(A_EXAMPLE + 3)`.

(maybe should change in `python/qemu/ot/otp/descriptor.py`)